torch/csrc/jit/runtime/interpreter.h

```diff
@@ -66,7 +67,7 @@ struct TORCH_API Code {
 };
 
 struct InterpreterState {
-  TORCH_API InterpreterState(const Code& code);
+  TORCH_API InterpreterState(const Code& code, TaskExecutor task_executor);
```

Comment:
maybe provide default value here too? then you don't need to update all unittests